guides/calculators/overview.md

```diff
@@ -0,0 +1,114 @@
+# Overview
+
+All calculators in Solidus inherit from the `Spree::Calculator` base class. They
+calculate and return the value of promotions, taxes, and shipping charges.
+Solidus includes built-in calculators for common types of calculations, such as
+flat rate shipping, percentage discounts, sales tax, and value-added tax (VAT).
+
+Whenever you create a new shipping method, tax rate, or promotion action, you
+also create a new instance of a `Spree::Calculator`.
+
+For example, if you create a new shipping method called "USPS Ground" that
+charges a flat rate of $10 USD, you would be creating an instance of the
+`Spree::Calculator::Shipping::FlatRate` calculator:
+
+- The new calculator instance would have `calculable_type` of
+  `Spree::ShippingMethod` (because it is associated with a shipping method, not a
```

Comment:
It would be best here that `calculable` is a polymorphic relationship and possibly link to the rails documentation on that.